src/main/scala/com/cognite/sdk/scala/common/K8sServiceToken.scala

```diff
@@ -0,0 +1,27 @@
+package com.cognite.sdk.scala.common
+
+import cats.effect.Async
+
+import java.nio.charset.Charset
+import java.nio.file.{Files, Paths}
+import scala.util.Try
+
+/** Only use for SessionProvider to interact with Cognite internal SessionAPI */
+trait K8sServiceTokenProvider {
+  def getKubernetesJwt[F[_]](implicit F: Async[F]): F[String]
+}
+
+object K8sServiceToken extends K8sServiceTokenProvider {
+  def getKubernetesJwt[F[_]](implicit F: Async[F]): F[String] = {
+    val serviceAccountTokenPath = "/var/run/secrets/tokens/cdf_token"
+    Try(
+      new String(Files.readAllBytes(Paths.get(serviceAccountTokenPath)), Charset.defaultCharset)
```

Comment:
Use fs2-io and
```scala
val serviceAccountTokenPath = Path("/var/run/secrets/tokens/cdf_token")
Files[F].readAll(serviceAccountTokenPath).through(fs2.text.utfs8.decode).compile.string
```
then use `.adaptError` if we want to translate exceptions to `SdkException`.

Reply:
Added fs2-io  into build.sbt and use it instead 👍